sdk/tables/aztable/entity_test.go

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package aztable
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func (s *tableClientLiveTests) TestAddBasicEntity() {
```

Comment:
Just curious if you benchmarked this against the current implementation.

Reply:
No I have not yet, but I will add an issue to add benchmarking in the future. [issue](https://github.com/Azure/azure-sdk-for-go/issues/15152).